### PR TITLE
Update user-facing terminology in the vtctld2 UI

### PR DESCRIPTION
**Pull request by doeg**
...because I'm a little bit cowardly of making logic changes to an unfamiliar UI, I figured it was less risky to _only_ change the display values and leave the Typescript alone (for now). Unfortunately, this leaves a handful of instances of oppressive terminology in the code base, but they won't be user-facing.

The static files and rice-box.go changes were auto-generated with the [new build tooling](https://github.com/vitessio/vitess/blob/master/web/vtctld2/README.md#building-for-production).

One thing worth noting is that this overwrites the files to go/vt/vtctld/rice-box.go that were made in https://github.com/vitessio/vitess/commit/1acb6e0ad79522ffe7867b33524b5d190b92ead5. Trying to parse that diff is an absolute nightmare, so I was unsuccessful in copying over the changes. 😞 I don't see any updates to the unbuilt `web/vtctld2/` files that might suggest what the changes were -- @deepthi do you happen to remember what they were, and if I should copy them over?

I also updated the copyright date in the new FE build scripts per @rohit-nayak-ps's suggestion on https://github.com/vitessio/vitess/pull/6473. :) 

Here's one example of the UI before vs. after:

Screen Shot 2020-07-27 at 1 27 05 PM
Screen Shot 2020-07-27 at 2 52 17 PM

### CODE REVIEW

**Comment by deepthi** [comment]
> 
> One thing worth noting is that this overwrites the files to go/vt/vtctld/rice-box.go that were made in [1acb6e0](https://github.com/vitessio/vitess/commit/1acb6e0ad79522ffe7867b33524b5d190b92ead5). Trying to parse that diff is an absolute nightmare, so I was unsuccessful in copying over the changes.  I don't see any updates to the unbuilt `web/vtctld2/` files that might suggest what the changes were -- @deepthi do you happen to remember what they were, and if I should copy them over?

I don't remember why I decided to run `make embed_static` on that PR, but you are correct that there were no actual UI changes. It is possible that there were earlier PRs that had been merged without updating `rice-box.go`. Since you have a working UI, what you have is now canonical :smile:

File: web/vtctld2/src/app/dashboard/shard.component.ts [hunk 1]
```diff
@@ -120,8 +120,8 @@ export class ShardComponent implements OnInit, OnDestroy {
     if (!isMaster) {
       // Add replication-related methods to the 'Change' section for slaves.
       result[1].items.push(
-        {label: 'Start Slave', command: (event) => {this.openStartSlaveDialog(); }},
-        {label: 'Stop Slave', command: (event) => {this.openStopSlaveDialog(); }},
+        {label: 'Start Replica', command: (event) => {this.openStartSlaveDialog(); }},
```
Comment (deepthi) [inline web/vtctld2/src/app/dashboard/shard.component.ts:123]:
```suggestion
        {label: 'Start Replication', command: (event) => {this.openStartSlaveDialog(); }},
```

File: web/vtctld2/src/app/dashboard/shard.component.ts [hunk 2]
```diff
@@ -120,8 +120,8 @@ export class ShardComponent implements OnInit, OnDestroy {
...
+        {label: 'Stop Replica', command: (event) => {this.openStopSlaveDialog(); }},
```
Comment (deepthi) [inline web/vtctld2/src/app/dashboard/shard.component.ts:124]:
```suggestion
        {label: 'Stop Replication', command: (event) => {this.openStopSlaveDialog(); }},
```

File: web/vtctld2/src/app/dashboard/tablet.component.ts [hunk 3]
```diff
@@ -116,19 +116,19 @@ export class TabletComponent implements OnInit, OnDestroy {
   }
 
   openStartSlaveDialog() {
-    this.dialogSettings = new DialogSettings('Start', `Start Slave, ${this.tablet.label}`, '',
-                                             `There was a problem starting slave, ${this.tablet.label}:`);
-    this.dialogSettings.setMessage(`Started Slave, ${this.tablet.label}`);
+    this.dialogSettings = new DialogSettings('Start', `Start Replica, ${this.tablet.label}`, '',
+                                             `There was a problem starting replica, ${this.tablet.label}:`);
```
Comment (deepthi) [inline web/vtctld2/src/app/dashboard/tablet.component.ts:120]:
```suggestion
                                             `There was a problem starting replication on, ${this.tablet.label}:`);
```

**Comment by doeg** [comment]
Sorry it took so long to apply your suggestions, @deepthi! I blame Docker + my terrible internet. 😂 

Here's what it looks like now:

Screen Shot 2020-07-27 at 4 39 52 PM
Screen Shot 2020-07-27 at 4 39 54 PM

It might be a good idea to squash merge this PR. 🎃 I've been committing the .ts file changes + the generated assets in separate commits to tidy up the commit history; thinking in terms of rollbacks + cherry picking, it probably makes the most sense to treat this PR as an atomic unit. Up to you though!

File: web/vtctld2/src/app/dashboard/shard.component.ts (hunk 1 above)
Comment (doeg) [reply web/vtctld2/src/app/dashboard/shard.component.ts]: Surprisingly, GitHub didn't auto-resolve these suggestions. :) I applied them manually so I didn't add them as individual commits. I'll go ahead and resolve them as I'm pretty sure I got 'em all!

**Comment by deepthi** [comment]
All tests are showing green in the Actions view.

**Pull request merged**